scopes/react/component-highlighter/component-highlighter/bit-react-component.ts

```diff
@@ -0,0 +1,5 @@
+import type { ComponentMetaHolder } from '@teambit/babel.bit-react-transformer';
+
+export function hasComponentMeta(component: any): component is ComponentMetaHolder {
```

Comment:
dont we have a type for `componentMeta` to use instead of `any`?

Reply:
cool, didnt know about the `is` keyword

Reply:
no, this is the point - we can an "any" object is identify it to be `ComponentMetaHolder`